cmd/loom/loom.go

```diff
@@ -547,6 +552,10 @@ func resetApp(cfg *config.Config) error {
 	return destroyDB(cfg.DBName, cfg.RootPath())
 }
 
+func resetEvm(cfg *config.Config) error {
```

Comment:
Not much point in having a one line function, just inline this code in the command implementation.